### ISSUE TITLE
Allow font-family to be customised using CSS variables

### ISSUE THREAD

**Issue by robcaldecottvelo** [material-components/material-components-web#5003]
Currently there is no way to override the `font-family` for some components (like `button`) without using:

- Custom CSS rules.
- The SASS `$mdc-theme-font-family` variable.

However, we can update the various theme colours using CSS variables which is simple and allows for easy custom theming (see the RMWC web site for an example.)

It would be great if the main theme `font-family` could also be overridden using a CSS variable in the same way as the various colours, e.g.:

```css
:root {
  --mdc-theme-primary: #6200ee;
  --mdc-theme-secondary: #018786;
  --mdc-theme-font-family: "Custom Font Name"
}
```

**Comment by abhiomkar** [comment]
This is related to #4709

PRs welcome! Thanks.

**Comment by asyncLiz** [comment]
Fixed in #5618
